### ISSUE TITLE
Unable to login with HTTPS custom domain URL

### ISSUE THREAD

**Issue by kkhamruddin-pk** [PaladinCloud/CE#494]
I have enabled HTTPS for the ALB with a custom domain. I have allowed https to whole world for paladin security group that is attached to ALB. I am able to access the URL but failing to login.

<img width="347" alt="image" src="https://user-images.githubusercontent.com/33874243/192977900-4e1140c6-1df0-4aeb-9c61-52b4cd5b2333.png">

Please suggest

**Comment by kkhamruddin-pk** [comment]
@rnithinpaladin : please let me know your availability so that we can connect and I can walk through the issue.

I tried allowing via VPN security group as well but still no luck

**Comment by rnithinpaladin** [comment]
HI @kkhamruddin-pk 

can we have call at 4.00 pm IST ?

**Comment by kkhamruddin-pk** [comment]
Sure @rnithinpaladin. Please send me the invite.

**Comment by kkhamruddin-pk** [comment]
Thanks for connecting @rnithinpaladin. Below is the summary our call.

Initial setup needs a security group rule for HTTP to be allowed to whole world in order to login to the pcloud portal. As this is not recommended, we have enabled HTTPS for the ALB with SSL Certificate for custom domain stored in AWS Certificate Manager. And updated security group of ALB and ECS containers to allow HTTPS traffic from OpenVPN security group only.

We are able to access the URL with HTTPs but we were unable to login to the portal. Upon troubleshooting want I found was the ECS containers configs are calling the external ALB with in the application code. Because of this, the login request is not passing via openVPN and getting authentication issues.

To avoid this, we suggest you to update the ECS container configurations to point to ECS container services DNS names in order to communicate each paladin cloud applications each other internally. Probably they have to use service discovery for ECS containers for their applications. 

<img width="479" alt="image" src="https://user-images.githubusercontent.com/33874243/193038305-237ba4a6-4758-42ee-a748-130a4c6d2b9e.png">

For time being until you fix the issue, we need to allow HTTP to whole world in order to fix issues with ECS container communicating each other. The pcloud portal still can be accessed with HTTPS.
